base_name_search_improved/tests/test_name_search.py

```diff
@@ -28,6 +28,22 @@ def setUp(self):
              'supplier': True,
              'phone': '+351 777 333 555'})
 
+    def test_default_name_search(self):
+        """ Check if default name_search is preserved """
+        # pre-8.0.1.0.2 name_search_use_standard was set to NULL instead of
+        # True for any models initialized after this module, thereby breaking
+        # default name_search functionality (#1389).
+        # - Test for no NULL values
+        self.env.cr.execute(
+            'select count(*) from ir_model where '
+            'name_search_use_standard is null'
+        )
+        self.assertEquals(self.env.cr.fetchone()[0], 0)
+        # - report (auto)installs after base, so test if default name_search
+        # works well for one of its models
+        self.assertEquals(len(self.env['report.paperformat'].name_search(
+            'US Letter', operator='=')), 1)
```

Comment:
this might break depending of the module available or not. Better just create a new `ir.model` record via SQL and assert the value of the field